include/sound/soc-acpi.h

```diff
@@ -104,6 +104,7 @@ struct snd_soc_acpi_adr_device {
 	const u8 num_endpoints;
 	const struct snd_soc_acpi_endpoint *endpoints;
 	const char *name_prefix;
+	const char dai_name[64];
```

Comment:
I don't see the logic @bardliao. There should be an array of dais for a codec device. Why would we allow for a single dai_name at the codec level?

Reply:
> I don't see the logic @bardliao. There should be an array of dais for a codec device. Why would we allow for a single dai_name at the codec level?

Having an array of dais for a codec device was my option 1, but I found it is difficult to implement based on the existing sof_sdw.c. That's why I chose option 2. i.e. Allow single dai for a codec device and use different devices when we want to use different dais. I will revisit option 1 if it is the preferred way.